packages/documentation/src/pages/getting-started/bundles-and-code-organization.mdx

```diff
@@ -0,0 +1,86 @@
+# Bundle and Code Organization
+
+## Files Overview
+
+- `/config`
+  - jest configs - proof of concepts- not used in CI
+    - `jest-integration.config.js`
+    - `jest-vrt.config.js`
+  - Nightwatch e2e test runner and configurations
+    - `nightwatch-globals-sauce.js`
+    - `nightwatch-sauce.js`
+    - `nightwatch.docker-compose.js`
+    - `nightwatch.js`
+  - sass config: `sass-lint.yml`
+  - webpack production and dev configs
+    - `webpack.config.js`
+    - `webpack.dev.config.js`
+  - build setup script
+    - `manifest-loader.js` - passes front matter of content files to app entry files
+	  - `parse-app-settings.js` - reads settings.js - a file containing app information generated during Metalsmith build
+- `/docs` - best practices and troubleshooting docs
+- `/jenkins` - jenkins setup, build, deploy script for `vets-websites` repo
+- `/script`
+  - `/custom-eslint` - project specific linting plugin
+  - `/hooks` - -git -precommit hook
+  - `app-list.sh` - bash script that outputs a list of the applications in vets-website
+  - `build.js` - imports and runs build script in src/site/stages/build
+  - `csv-path-builder.js` - TODO
+  - `heroku-postbuild.sh` - TODO - content deploy to heroku
+  - `mocha.js` - unit test runner and watch task
+  - `prearchive.js` - imports and run prearchive in src/site/stages/prearchive TODO
+  - `prebuild.js` - npm prebuild script that checks node version and install precommit hooks
+  - `preview.js` - metalsmith build script for drupal preview
```

Comment:
Might be better described as "Express server used to preview content pages on demand from the Drupal admin site"